tone-analyzer/src/main/java/com/ibm/watson/developer_cloud/tone_analyzer/v3/model/ToneChatInput.java

```diff
@@ -0,0 +1,30 @@
+package com.ibm.watson.developer_cloud.tone_analyzer.v3.model;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.List;
+
+/**
+ * This object represents the input for Tone Chat analysis. It contains a list of utterances.
+ */
+public class ToneChatInput {
```

Comment:
Rename this to be `ToneChatOptions` and make sure it implements the `Builder` pattern. You can look at `RecognizeOptions` like in the Speech to Text API